composer.json

```diff
@@ -16,6 +16,7 @@
         "symfony/flex": "^2",
         "symfony/framework-bundle": "6.2.*",
         "symfony/http-client": "6.2.*",
+        "symfony/http-kernel": "6.2.*",
```

Comment:
To w sumie jest już zawarte w symfony/framework-bundle - ale dodanie tego osobno to nie jest też błąd więc luz. Jako dowód że to już jest w zależnościach framework-bundle, obczaj sobie tego composer.json:

https://github.com/symfony/framework-bundle/blob/6.2/composer.json

Widać to też tutaj:

https://packagist.org/packages/symfony/framework-bundle

Btw: packagist to jest oficjalne repozytorium / rejestr / **registry** paczek do composera